src/index.ts

```diff
@@ -41,6 +42,9 @@ i18next.init({
 		"cs-CZ": {
 			translation: csCZ,
 		},
+		"it-IT": {
+			translation: itIT,
+		}
```

Comment:
Could be worth looking into generifying this so that it automatically gets all the language files instead of needing to manually add them here? Same for BulbBotUtils

Reply:
Yea